spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.spec.js

```diff
@@ -242,14 +242,23 @@ describe('Google Tag Manager page view tracking', function () {
       document.body.removeChild(content)
     })
 
-    it('set correctly', function () {
+    it('set correctly using the #content element', function () {
       content.setAttribute('lang', 'wakandan')
       expected.page_view.language = 'wakandan'
 
       GOVUK.analyticsGa4.analyticsModules.PageViewTracker.init()
       expect(window.dataLayer[0]).toEqual(expected)
     })
 
+    it('set correctly using the html lang attribute', function () {
+      document.documentElement.setAttribute('lang', 'cy')
```

Comment:
Yeah okay this doesn't work. I nearly got it to work with a `setTimeout` but the expectation then happened before it, so it didn't work, and you can't include the expects in a timeout as they don't get evaluated properly. You can remove this bit, but maybe make a note somewhere that setting the lang on the HTML element in Jasmine and therefore testing it isn't possible.